Marlin/pins_PRINTRBOARD.h

```diff
@@ -121,7 +121,8 @@
 #endif // ULTRA_LCD && NEWPANEL
 
 #if ENABLED(VIKI2) || ENABLED(miniVIKI)
-  #define BEEPER_PIN 32 //FastIO
+  //FastIO
+  #define BEEPER_PIN 32
```

Comment:
This is better as an end-of-line comment. There it says "this pin number comes from FastIO" whereas placing it above the line says "The following line is your FastIO."